src/sidebar/components/tag-editor.js

```diff
@@ -17,23 +17,38 @@ let datalistIdCounter = 0;
 function TagEditor({ onEditTags, tags: tagsService, tagList }) {
   const inputEl = useRef(null);
   const [showSuggestions, setShowSuggestions] = useState(false);
+  const [suggestionsFiltered, setSuggestionsFiltered] = useState([]); // For Chrome only
```

Comment:
`suggestionsFiltered` sounds like a boolean to me (as in "_are_ the suggestions filtered?"). `filteredSuggestions` would have sounded more like a list.

Reply:
Its been a week so my memory is hazy with this, but I'm pretty sure I trie to stub `navigator` this w/o success, but I'll give it a try again

Reply:
Its been a week so my memory is hazy with this, but I'm pretty sure I trie to stub `navigator` this w/o success, but I'll give it a try again

Reply:
I think this reply was meant for a different comment.